content/schemas/sign_request_create_signer.yml

```diff
@@ -0,0 +1,112 @@
+---
+title: Signer fields for Create Sign Request
+
+type: object
+
+description: |-
+  The schema for a Signer in a POST Sign Request request body
+
+required:
+  - email
+
+properties:
+  email:
+    type: string
+    description: Email address for signer
+    example: example@gmail.com
+
+  first_name:
+    type: string
+    description: First name of signer
+    example: Aaron
+
+  last_name:
+    type: string
+    description: Last name of signer
+    example: Levie
+
+  role:
+    type: string
+    enum:
+      - signer
+      - approver
+      - subscriber
+      - final_copy_reader
+    description: Defines the role of the signer in the sign request. Signers will
+      need to sign the document, Approvers may just approve the document,
+      and subscribers will not need to take any actions but will be notified
+      of each action. Finally, final_copy_reader will only receive the finished
+      sign request with a sign log
+    example: signer
+
+  is_in_person:
+    type: boolean
+    description: 'When used in combination with an embed url on the sender,
+      after sender has signed, they will be redirected to the next in_person
+      signer, see: In person signing'
+    example: true
+
+  order:
+    type: integer
+    description: Order of signer
```

Comment:
Is this needed, since the signers will be found in an array which is naturally ordered?

Reply:
Signing order, can be duplicates